Main.java

```diff
@@ -0,0 +1,45 @@
+package ListLab;
+
+import java.util.Iterator;
+
+public class Main{
+	public static void main(String[] list){
+		MyLinkedList<Integer> linkedList = new MyLinkedList<>();
+
+		fillList(linkedList);
+		printList(linkedList);
+		reverseList(linkedList);
+		printList(linkedList);
+	}
+
+	public static void fillList(MyLinkedList<Integer> list){
+		for (int i = 0; i <= 20; i++){
+			list.add(i)
+		}
+	}
+
+	public static void printList(MyLinkedList<Integer> list) {
+		System.out.printlm("Printing LinkedList..................");
+
+		for (int i = 0; i < list.size; i++) {
+			System.out.println(list.get(i))
+		}
+
+		System.out.println(".........Finished printing LinkedList")
+	}
+
+	public static MyLinkedList<Integer> reverseList(MyLinkedList<Integer> list){
+		MyLinkedList<Integer> newList = new MyLinkedList<>();
```

Comment:
One of the requirements was "The method should reverse the list in place, meaning you will modify the list directly without making a whole new copy of the list." You're moving nodes over one by one, not all at once, but still you're copying the whole list.